x-pack/plugins/maps/public/connected_components/add_layer_panel/flyout_body/layer_wizard_select.tsx

```diff
@@ -165,7 +166,8 @@ export class LayerWizardSelect extends Component<Props, State> {
 
     return (
       <>
-        {this._renderCategoryFacets()}
+        <EuiFlexItem> {this._renderCategoryFacets()}</EuiFlexItem>
```

Comment:
I did not know EuiFlexItem could be used outside of EuiFlexGroup. What does EuiFlexItem provide? I can not see an visual difference when wrapping `this._renderCategoryFacets()` in EuiFlexItem

Reply:
Thanks for noticing this. There's a parent `EuiFlexGroup` so the `EuFlexItem` was being used inside of it. But this is actually not doing anything. It was an attempt to remove the horizontal scrollbar but then I found what was causing that and this `EuFlexItem` stayed forgotten.